Objection.xcodeproj/project.pbxproj

```diff
@@ -1285,6 +1311,7 @@
 					"-lstdc++",
 					"-all_load",
 				);
+				PRODUCT_BUNDLE_IDENTIFIER = "atomicobject.${PRODUCT_NAME:rfc1034identifier}";
```

Comment:
It's weird that this is in many places in this file instead of just in the project settings.

Reply:
🤷‍♂ what do you want to do?